src/basic_recipes/streamplot.jl

```diff
@@ -28,7 +28,7 @@ See the function `Makie.streamplot_impl` for implementation details.
         maxsteps = 500,
         color = norm,
 
-        arrow_size = 15,
+        arrow_size = nothing,
```

Comment:
This should be `automatic` since we generally use that instead of nothing when things get derived later

Reply:
OK, I've changed `nothing` to `automatic`!